pkg/controller/statics/statics_controller.go

```diff
@@ -153,5 +151,6 @@ func deleteSCC(logger logr.Logger, client crclient.Client) {
 	logger.Info("Manually deleting SecurityContextConstraints. See https://github.com/openshift/aws-efs-operator/issues/23")
 	scce := findStatic(types.NamespacedName{Name: sccName})
 	// Delete() does the logging. We're ignoring any errors.
-	scce.Delete(logger, client)
+	err := scce.Delete(logger, client)
+	logger.Error(err, "Error in manually deleting SecurityContextConstraints")
```

Comment:
Per the comment, logging again is redundant. Recommend explicitly ignoring the error:
```suggestion
	_ := scce.Delete(logger, client)
```